examples/hello.rs

```diff
@@ -1,17 +1,44 @@
-use std::error::Error;
 use wgpu_glyph::{ab_glyph, GlyphBrushBuilder, Section, Text};
+use winit::{
+    event_loop::EventLoop,
+    window::Window,
+};
 
-fn main() -> Result<(), Box<dyn Error>> {
-    env_logger::init();
-
+fn main() {
     // Open window and create a surface
     let event_loop = winit::event_loop::EventLoop::new();
 
     let window = winit::window::WindowBuilder::new()
         .with_resizable(false)
+        .with_title("Hello wgpu glyph !")
         .build(&event_loop)
         .unwrap();
 
+    #[cfg(not(target_arch = "wasm32"))]
+    {
+        wgpu_subscriber::initialize_default_subscriber(None);
+        // Temporarily avoid srgb formats for the swapchain on the web
+        pollster::block_on(run(event_loop, window));
+    }
+    #[cfg(target_arch = "wasm32")]
+    {
+        std::panic::set_hook(Box::new(console_error_panic_hook::hook));
+        console_log::init().expect("could not initialize logger");
+        use winit::platform::web::WindowExtWebSys;
+        // On wasm, append the canvas to the document body
+        web_sys::window()
+            .and_then(|win| win.document())
+            .and_then(|doc| doc.body())
+            .and_then(|body| {
+                body.append_child(&web_sys::Element::from(window.canvas()))
+                    .ok()
+            })
+            .expect("couldn't append canvas to document body");
+        wasm_bindgen_futures::spawn_local(run(event_loop, window));
+    }
+}
+
+async fn run(event_loop: EventLoop<()>, window: Window) {
```

Comment:
Why have we introduced `async` at the top level here?

It looks like we are not awaiting anywhere. In fact, we are running `block_on` inside a `Future` which is forbidden by contract.

Reply:
`async` is required for `wasm_bindgen_futures::spawn_local`, it's inspired from [Wgpu-rs Hello triangle](https://github.com/gfx-rs/wgpu-rs/tree/master/examples/hello-triangle) example.
So I have remove the `futures::executor::block_on` that's not necessary anymore.